[SignIn] Adding appliedEventListeners to beta Prod, MC and FF. (#18264)
microsoftgraph/microsoft-graph-docs: api-reference/beta/resources/signin.md

|appDisplayName|String|The application name displayed in the Azure Portal. Supports `$filter` (`eq` and `startsWith` operators only).|
|appId|String|The application identifier in Azure Active Directory. Supports `$filter` (`eq` operator only).|
|appliedConditionalAccessPolicies|[appliedConditionalAccessPolicy](appliedconditionalaccesspolicy.md) collection|A list of conditional access policies that are triggered by the corresponding sign-in activity.|

> |appliedEventListeners|[appliedAuthenticationEventListener](../resources/appliedauthenticationeventlistener.md) collection|Detailed information about the applied event listeners or listeners that are triggered by the corresponding events in an authentication activity.|

Reviewer: ```suggestion
|appliedEventListeners|[appliedAuthenticationEventListener](../resources/appliedauthenticationeventlistener.md) collection|Detailed information about the applied event listeners or listeners that were triggered by the corresponding events in the sign-in event.|
```

Reviewer: @junrlin does this change make sense? I assume the authn activity is always the sign-in event represented by the signin resource

Author: I'd like to say we will also have events related to sign-up process, so I would keep the old sentence to describe more.

> |appliedEventListeners|[appliedAuthenticationEventListener](../resources/appliedauthenticationeventlistener.md) collection|Detailed information about the applied event listeners or listeners that were triggered by the corresponding events in an authentication activity.|

Author: Not limited to Azure Logic Apps, it can be Azure Functions or other ways to do the same functionality.

> |appliedEventListeners|[appliedAuthenticationEventListener](../resources/appliedauthenticationeventlistener.md) collection|Detailed information about the applied event listeners or listeners that were triggered by the corresponding events in an authentication activity.|

Reviewer 2: That's OK. "such as" shows it's only just Azure Logic Apps. You can also edit the sentence to "such as Azure Logic Apps and Azure Functions". The intent is to show customers what listeners are supported.

> |appliedEventListeners|[appliedAuthenticationEventListener](../resources/appliedauthenticationeventlistener.md) collection|Detailed information about the applied event listeners or listeners that were triggered by the corresponding events in an authentication activity.|

Author: Got it, thanks!
